tools/slang-test/slang-test.vcxproj.filters

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These diffs at the start of the VS files have typically come up from the files being edited manually rather than generated using premake. If possible we should stick to using premake to generate the files whenever adding new sources to the project, but it isn't a big deal.